docs/integrations_gitlabci.md

```diff
@@ -87,3 +87,30 @@ kics-scan:
 <br>
 
 <img src="https://raw.githubusercontent.com/Checkmarx/kics/master/docs/img/kics_gitlab_pipeline_sast_report_result.png" width="850">
+
+## Code Quality integration
```

Comment:
Can you include the Code Quality integration in `kics\docs\integrations.md` and `kics\mkdocs.yml`, please?